### PR TITLE
Add CLI Guide UI components

### PR DESCRIPTION
**Pull request by axbarsan**
Towards https://github.com/giantswarm/giantswarm/issues/16564

This PR adds the UI components necessary for creating CLI Guides (see issue for more info). The actual CLI guides will be implemented in a separate PR.

## Preview

<details>
<summary>Closed</summary>

image

</details>

<details>
<summary>Open</summary>

image

</details>

<details>
<summary>Open (with additional information)</summary>

image

</details>

### CODE REVIEW

File: src/components/UI/Display/MAPI/CLIGuide/CLIGuideAdditionalInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  ...props
+}) => {
+  return (
+    <Box {...props}>
+      <Box>
+        <Heading level={5}>Additional information</Heading>
+      </Box>
+      <Box>
+        {links?.map((link) => (
+          // eslint-disable-next-line react/jsx-no-target-blank
```
Comment (axbarsan) [inline src/components/UI/Display/MAPI/CLIGuide/CLIGuideAdditionalInfo.tsx:32]: This is a false positive, we provide `noopener noreferrer` when `_blank` is used.

File: src/components/UI/Display/MAPI/CLIGuide/index.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  ThemeType,
+} from 'grommet';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { css } from 'styled-components';
+
+const customTheme: ThemeType = {
+  global: {
+    colors: {
+      text: 'text-weak',
```
Comment (axbarsan) [inline src/components/UI/Display/MAPI/CLIGuide/index.tsx:16]: Override text color in the accordion 🤯

File: src/styles/theme.ts [hunk 3]
```diff
@@ -131,8 +131,8 @@ const theme = deepMerge(generate(16), {
         light: '#375b70',
       },
       'background-contrast': {
-        dark: '#15253111',
-        light: '#15253111',
+        dark: '#15253150',
+        light: '#15253150',
```
Comment (marians) [inline src/styles/theme.ts:135]: Assuming that's the main container background color, can we please use [darkBlueDarker6 (#10212c)](https://github.com/giantswarm/happa/blob/master/src/styles/theme.ts#L26)?

Comment (marians) [reply src/styles/theme.ts:135]: BTW, I didn't intend to make all accordions use that background color. Just the one used for this purpose.

Comment (axbarsan) [reply src/styles/theme.ts:135]: Updated screenshots.

**Review by marians — APPROVED** [review]
Looking forward to having it in use!

**Pull request merged**